lib/tasks/earthworks.rake

```diff
@@ -59,4 +59,25 @@ namespace :earthworks do
       File.open(fn, 'w') {|f| f << JSON.pretty_generate(data) }
     end
   end
+  namespace :geomonitor do
+    desc 'Update the layers GeoMonitor checks from the Solr index'
+    task update: [:environment] do
+      response = Blacklight.default_index.connection.get(
+        'select',
+        params: { q: '*:*', fl: '*', rows: 999_999 }
```

Comment:
Does Blacklight/RSolr provide a paging API that we could use instead to downloading all data in one query?

Reply:
Great suggestion, I'll look into it.